### ISSUE TITLE
feat: alert user that unpaywall API uses email address from git settings

### ISSUE THREAD

**Issue by blairw** [CoLRev-Ecosystem/colrev#127]
# Feature Request

**Describe the Feature Request**

It would be good if the documentation and/or colrev CLI tools alerts the user that the unpaywall API call will automatically deduce the user's email address from their Git settings, in case the user would like to use a different email address for interacting with the unpaywall API.

If changing some key in `settings.json` (possibly `project.authors[0].email`? or some new key, e.g., `email_address_for_unpaywall_api`?) makes it possible for the user to use a different email address, it would be good for the user to know this.

**Describe Preferred Solution**

```
CoLRev has determined from your Git settings that your username is: alice.bob@example.com
If you would like to use a different email address, please modify settings.json key email_address_for_unpaywall_api

More instructions online at: <link to documentation goes here>
```

**Describe Alternatives**

Perhaps: user is asked for their email address for the unpaywall API during initial setup?

**Related Code**

- `get_name_mail_from_git()` is defined in `colrev/env/environment_manager.py`
- email address is appended to unpaywall API call in `colrev/ops/built_in/pdf_get/unpaywall.py`:

```
source = (
  f"https://api.unpaywall.org/v2/{record.data['doi']}"
  + f"?email={pdf_get_operation.review_manager.email}"
)
```

**Additional Context**

N/A

**If the feature request is approved, would you be willing to submit a PR?**

Yes

**Comment by geritwagner** [comment]
That's right - we should notify users about the email used for unpaywall and provide an option to select a custom email address. The field could be stored in a local settings file (not the project settings because different team members may use different emails or API credentials for other extensions). The local settings should be implemented as part of v0.10.0.

**Comment by geritwagner** [comment]
Note : the same issue applies to a number of SearchSources (e.g., crossref, dblp, pubmed).

**Comment by tmahmood** [comment]
@all-contributors please add @geritwagner for Code, Documentation, Data, Content

**Comment by tmahmood** [comment]
@all-contributors please add @geritwagner, @julianprester for Code, Documentation, Data, Content

**Comment by tmahmood** [comment]
@all-contributors please add @geritwagner for Code, Documentation, Data, Content

@all-contributors please add @julianprester for Code, Documentation, Data, Content
